Open Judge System/Web/OJS.Web/Areas/Contests/Controllers/ResultsController.cs

```diff
@@ -487,41 +509,77 @@ public ActionResult StatsChart(int contestId)
 
             var participants = this.participantsData
                 .GetAllByContestAndIsOfficial(contest.Id, official)
-                .AsNoTracking();
+                .ToList()
+                .OrderByDescending(p => p.TotalScoreSnapshot);
```

Comment:
Try not to get all the participants at once in memory, where not necessary (results pages).

Reply:
Done